docs/pages/components/field/api/field.js

```diff
@@ -69,7 +69,7 @@ export default [
             },
             {
                 name: '<code>horizontal</code>',
-                description: 'Group label and control on the same line for horizontal forms',
+                description: 'Group label and control on the same line for horizontal forms. Do not mix with <code>grouped</code>.',
```

Comment:
Is there a more clear way to say this? And should we also explain, in a simple form, why we shouldn't mix them?

Reply:
I briefly explained the reason why we should not mix them. However, there would not be a reasonable explanation because it is a design flaw we have to address.

Reply:
DM over discord. Maybe we could discuss this flaw over the weekend?